dvc/env.py

```diff
@@ -7,3 +7,5 @@
 DVCLIVE_HTML = "DVCLIVE_HTML"
 DVCLIVE_RESUME = "DVCLIVE_RESUME"
 DVC_IGNORE_ISATTY = "DVC_IGNORE_ISATTY"
+DVC_EXP_GIT_REMOTE = "DVC_EXP_GIT_REMOTE"
+DVC_EXP_CHECKPOINT_PUSH = "DVC_EXP_CHECKPOINT_PUSH"
```

Comment:
I think this should be named something more like `DVC_EXP_AUTO_PUSH`

It is not specific to checkpoints, and should not have `CHECKPOINTS` in the name.

Reply:
Yes, because we had added auto push after the running result being committed. Other experiments have also been affected.